lighthouse-core/scripts/pptr-run-devtools.js

```diff
@@ -316,7 +317,7 @@ async function run() {
   }
 
   const browser = await puppeteer.launch({
-    executablePath: process.env.CHROME_PATH,
+    executablePath: getChromePath(),
```

Comment:
This still defaults to `CHROME_PATH` but is the one I felt less sure about. Let me know if anyone would like it the other way and I can always revert.